### DISCUSSION TITLE
Send us your GitHub username

### DISCUSSION THREAD

**Discussion by eculler** [cu-esiil-edu/2024-data-short-course#8] [Announcements]
[Create an issue](https://github.com/cu-esiil-edu/2024-data-short-course/issues/new?assignees=eculler&labels=add+to+team&projects=&template=add-to-team.md&title=Add+to+short+course+student+team) with your GitHub username in it so we can add you to the Data Short Course Students Team!

**Comment by DataAnalyticsUzb** [comment]
Shakhzod Mirzaev

Github username: @DataAnalyticsUzb 

Please, add me to Github community of ESIIL Data Short Course 2024!

@eculler

**Comment by caroarguiano** [comment]
Carolina Arguiano

Github username: @caroarguiano

Please, add me to Github community of ESIIL Data Short Course 2024. Thank you!

**Comment by smalljackcodes** [comment]
Github username: @smalljackcodes

Please, add me to Github community of ESIIL Data Short Course 2024! Thank You. 

@eculler

**Comment by nadiacharki** [comment]
Nadia CHARKI

Github username: @nadiacharki

Please, add me to GitHub community of ESIIL Data Short Course 2024. 

Thank you!

@eculler

**Comment by wasanRuntime** [comment]
Andrew Simiyu

Github username: @wasanRuntime

Please, add me to GitHub community of ESIIL Data Short Course 2024.

Thank you!

**Comment by DharaniSureshBabu** [comment]
Dharani Suresh Babu 

@DharaniSureshBabu. Thank you for adding my name.

**Comment by Sordhiq** [comment]
Name: Sodiq Jinad

Username: sordhiq.github.io

@eculler kindly get me added to the github community of the ESIIL Data Short Course 2024

**Comment by yamlakyam** [comment]
Name: Yamlak Asrat Bogale
Github username: @yamlakyam

Please, add me to GitHub community of ESIIL Data Short Course 2024. Thank you.

@eculler
